notify/docs/listing-templates/list.java

```diff
@@ -0,0 +1,3 @@
+TemplateList allTemplates = client.getAllTemplates(null);
```

Comment:
You wrote above "If omitted, the method returns all templates", but then you're passing `null` in the example?

Reply:
I've reworded general doc that accompanies this